### PR TITLE
[Documentation][CustomizationGuide] Fixes

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/customization/factory.rst [hunk 1]
```diff
@@ -102,9 +101,10 @@ as a decorating service in the ``app/Resources/config/services.yml``.
             arguments: ['@app.factory.product.inner']
             public: false
 
-3.You can use the new method of factory in routing.
+**3.** You can use the new method of factory in routing.
```
Comment (TheMadeleine) [inline docs/customization/factory.rst]: of **the** factory

File: docs/customization/grid.rst [hunk 2]
```diff
@@ -102,10 +102,10 @@ If you would like to disable some actions in any grid, you just need to set its
                             type: delete
                             enabled: false
 
-How to modify an action of a grid?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How to add an action of a grid?
```
Comment (TheMadeleine) [inline docs/customization/grid.rst]: well... nope. We describe adding actions in the bundle docs. Here we need to have the modification. Not adding.

I know we've discussed this, but still we need this example here. You can change it to some inexistent grid modification (like `app_supplier`).

File: docs/customization/menu.rst [hunk 3]
```diff
@@ -205,31 +189,23 @@ type is default to make the example easily customizable.
...
-        - { resource: "services.yml" }
+                - { name: kernel.event_listener, event: sylius.menu.admin.customer.show, method: addAdminCustomerShowMenuItems }
 
 How to customize Admin Order Show Menu?
 ---------------------------------------
 
 .. tip::
 
-    Admin order show menu is the set of buttons (currently only ``Cancel`` button) in the right top corner on the ``/admin/orders/{id}`` url.
+    Admin order show menu is the set of buttons in the right top corner on the ``/admin/orders/{id}`` url.
```
Comment (TheMadeleine) [inline docs/customization/menu.rst]: Why are we changing this?

File: docs/customization/state_machine.rst [hunk 4]
```diff
@@ -21,7 +21,7 @@ How to customize a State Machine?
 How to add a new state?
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would like to add a new **state** to the state machine of :doc:`Order </book/orders/orders>`.
+Let's assume that you would like to add a new **state** to the state machine of an :doc:`Order </book/orders/orders>`.
```
Comment (TheMadeleine) [inline docs/customization/state_machine.rst]: ```:doc:`the Order state machine </book/orders/orders>`.```

File: docs/customization/state_machine.rst [hunk 5]
```diff
@@ -34,10 +34,15 @@ You will need to add these few lines to the ``state_machine.yml``:
...
+.. tip::
+
+    Run ``$ php bin/console debug:winzou:state-machine sylius_order``
+    to check if the state machine has changed to your implementation.
+
 How to add a new transition?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would like to add a new **transition** to the state machine of :doc:`Order </book/orders/orders>`,
+Let's assume that you would like to add a new **transition** to the state machine of an :doc:`Order </book/orders/orders>`,
```
Comment (TheMadeleine) [inline docs/customization/state_machine.rst]: ```:doc:`the Order state machine </book/orders/orders>`.```

File: docs/customization/state_machine.rst [hunk 6]
```diff
@@ -65,7 +75,7 @@ How to remove a state and its transitions?
 How to add a new callback?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would like to add a new **callback** to the state machine of :doc:`Order </book/orders/orders>`,
+Let's assume that you would like to add a new **callback** to the state machine of an :doc:`Order </book/orders/orders>`,
```
Comment (TheMadeleine) [inline docs/customization/state_machine.rst]: ```:doc:`the Order state machine </book/orders/orders>`.```

File: docs/customization/template.rst [hunk 7]
```diff
@@ -26,24 +26,24 @@ How to customize templates?
...
+    {% import '@SyliusUi/Macro/messages.html.twig' as messages %}
 
     {% block content %}
     <div class="ui column stackable center page grid">
         {% if last_error %}
             {{ messages.error(last_error.messageKey|trans(last_error.messageData, 'security')) }}
         {% endif %}
 
-        // You can add a headline for instance to see if you are changing things in the correct place.
+        {#You can add a headline for instance to see if you are changing things in the correct place.#}
```
Comment (TheMadeleine) [inline docs/customization/template.rst]: Missing spaces.

File: docs/customization/template.rst [hunk 8]
```diff
@@ -81,14 +85,18 @@ Copy the contents of the original template to make your work easier. And then mo
     </div>
     <div class="ui segment">
 
-        // You can add a headline for instance to see if you are changing things in the correct place.
+        {#You can add a headline for instance to see if you are changing things in the correct place.#}
```
Comment (TheMadeleine) [inline docs/customization/template.rst]: Missing spaces

File: docs/customization/menu.rst (hunk 3 above)
Comment (GSadee) [inline docs/customization/menu.rst]: Because we have also a history button there

**Comment by pjedrzejewski** [comment]
Thanks Grzesiu!

**Pull request merged**